canonical-web-and-design/ubuntu.com
# Build openstack-cheat-sheet page

## Done

- Build page on `/openstack/openstack-cheat-sheet`
- Update "OpenStack cheat sheet" button on `/openstack/tutorials` to point to `/openstack/openstack-cheat-sheet` 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/openstack-cheat-sheet
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please check page against [copy doc](https://docs.google.com/document/d/1q37DrtjZhcgRK4NuL3CUsCv1aU6AQDX5uqtQ4ndVT-A/edit#)
- Please QA at https://ubuntu-com-11175.demos.haus/openstack/tutorials, https://ubuntu-com-11175.demos.haus/openstack/what-is-openstack and https://ubuntu-com-11175.demos.haus/openstack/openstack-cheat-sheet

## Issue / Card

Fixes [#4773](https://github.com/canonical-web-and-design/web-squad/issues/4773)


## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
